homeassistant/components/renault/renault_vehicle.py

```diff
@@ -201,7 +201,18 @@ async def set_hvac_schedules(
     @with_error_wrapping
     async def get_charging_settings(self) -> models.KamereonVehicleChargingSettingsData:
         """Get vehicle charging settings."""
-        return await self._vehicle.get_charging_settings()
+        full_endpoint = await self._vehicle.get_full_endpoint("charging-settings")
```

Comment:
`get_charging_settings` helper has been removed in the library
This replicates the old library method
It will be refactored completely in a follow-up PR